modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -10,10 +10,11 @@ class PowerOfAttorney::RequestController < ClaimsApi::V2::Veterans::PowerOfAttor
         FORM_NUMBER = 'POA_REQUEST'
 
         def request_representative
+          # validate target veteran exists
+          target_veteran
+
           poa_code = form_attributes.dig('poa', 'poaCode')
-          @claims_api_forms_validation_errors = validate_form_2122_and_2122a_submission_values(
-            target_veteran.participant_id, user_profile, poa_code
-          )
+          @claims_api_forms_validation_errors = validate_form_2122_and_2122a_submission_values(user_profile:)
```

Comment:
Revert [this change](https://github.com/department-of-veterans-affairs/vets-api/pull/18505/files#r1766246600). We can revisit the POA request controller in the POA requests tasks.